src/main/resources/templates/invalid-zipcode.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Order Failure</title>
+	<link rel="stylesheet" type="text/css" href="/css/layout.css"/>
+	<link rel="stylesheet" type="text/css" href="/css/style.css"/>
+</head>
+<body>
+	<div class="container">
+		<div class="message">
+			<h1>
+				Invalid Zipcode
+			</h1>
+			<p th:text="${error}">Please check your zipcode input try again, if this continues, please contact the pantry.</p>
```

Comment:
The `th:text="${error}"` on this line replace's the `<p>` tag's text with nothing because an `error` was not attached to the model.

Reply:
Thymeleaf artifact now removed.